boxyhq/jackson
# Directory sync: update raw user attributes

## What does this PR do?

This PR fixes the issue where the raw user attributes were not updated when the user was updated via the SCIM API.

This uses lodash `_.set` to update the raw user attributes.

## Type of change




## How should this be tested?



- Create a user via SCIM API
- Update any non standard attribute via SCIM API (For example - company, title, etc.)
- Update the attribute via SCIM API again
- Now the raw user attributes should have the updated value

## Checklist:
